.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,23 @@
+name: Run tests
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  publish-npm:
```

Comment:
I would change the name as it's a little bit confusing, since we're running tests in this job.

Reply:
Yep, copy-paste 👊